src/features/defi/providers/cosmos/components/CosmosManager/Overview/WithdrawCard.tsx

```diff
@@ -48,7 +51,7 @@ export const WithdrawCard = ({ asset, accountId: routeAccountId }: WithdrawCardP
 
   const undelegationEntries = useMemo(() => {
     if (!opportunityData) return []
-    if (isCosmosUserStaking(opportunityData) && opportunityData.undelegations.length) {
+    if (supportsUndelegations(opportunityData) && opportunityData.undelegations.length) {
```

Comment:
Now also applicable to FOXy, removes the cosmos terminology and makes this more generic